pkg/kubelet/config/file_linux_test.go

```
@@ -411,13 +411,11 @@ func writeFile(filename string, data []byte) error {
 	if err != nil {
 		return err
 	}
-	n, err := f.Write(data)
+	n, _ := f.Write(data)
```

Why ignoring the error of `f.Write`?